lively.halos/morph.js

```diff
@@ -1375,12 +1375,18 @@ class CopyHaloItem extends HaloItem {
     const { halo } = this; const { target } = halo; const world = halo.world();
     const isMultiSelection = target instanceof MultiSelectionTarget;
     halo.remove();
-
     connect(hand, 'update', this, 'update');
-
     if (isMultiSelection) {
       // FIXME! haaaaack
-      const copies = target.selectedMorphs.map(ea => world.addMorph(ea.copy()));
+      const copies = target.selectedMorphs.map(ea => {
```

Comment:
What does `ea` mean?

Reply:
That is a good question. We do not know, but this is how all iterating of selected Morphs in this method was done before we touched it :/ We just went with the convention.